rma_account/wizards/rma_add_account_move.py

```diff
@@ -122,9 +122,7 @@ def _get_existing_invoice_lines(self):
     def add_lines(self):
         rma_line_obj = self.env["rma.order.line"]
         existing_invoice_lines = self._get_existing_invoice_lines()
-        for line in self.line_ids.filtered(
-            lambda aml: aml.exclude_from_invoice_tab is False
-        ):
+        for line in self.line_ids.filtered(lambda l: l.product_id.id):
```

Comment:
`for line in self.line_ids.filtered(lambda aml: aml.display_type == "product"):` seems more reliable to me

Reply:
Done!